src/adapter/src/coord/controller_commands/parsed_state_updates.rs

```diff
@@ -41,7 +41,8 @@ pub enum ParsedStateUpdateKind {
         parsed_full_name: String,
     },
     TemporaryItem {
```

Comment:
Is there still any need to differentiate between `ParsedStateUpdateKind::Item` and `ParsedStateUpdateKind::TemporaryItem`?

Reply:
Maybe not, but I wanted to not blur the distinction in case there is a need. The code duplication doesn't seem too horrible, but ... :man_shrugging: